baremaps-openapi/src/main/resources/openapi.yaml

```diff
@@ -921,6 +1060,58 @@ components:
           nullable: true
         link:
           $ref: '#/components/schemas/link'
+    tileSet:
+      type: object
+      required:
+        - tilejson
+        - tiles
+      properties:
+        tilejson:
+          type: string
```

Comment:
Is it intended to keep null value in the json generated by the service? Jackson can be configured to ignore null values, but it is not always a good idea.

```
GET http://localhost:8080/tilesets/1 request 
{"tilejson":"test","name":null,"description":null,"version":null,"attribution":null,"template":null,"legend":null,"scheme":null,"tiles":[],"grids":[],"data":[],"minzoom":null,"maxzoom":null,"bounds":[],"center":[]}````

Reply:
It's not explicitly intended, we can ignore it to remove some noise i guess.